src/pretalx/urls.py

```diff
@@ -25,6 +25,7 @@
     path("orga/", include("pretalx.orga.urls", namespace="orga")),
     path("api/", include("pretalx.api.urls", namespace="api")),
     path("redirect/", redirect_view, name="redirect"),
+    path("goto/<code>", shortlink_view, name="shortlink"),
```

Comment:
I wonder if we should move this endpoint to `/redirect/goto/` or to `/api/goto/` instead – any top-level URL path restricts the choice of event shortcodes (and has the risk of an existing event on some instance conflicting with the chosen URL).

Reply:
`redirect/<code>` appears to be just fine - that is not in conflict with the `/redirect/?url=...` URLs